Volume/Supporting Views/TabBarReader.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  TabBarReader.swift
+//  Volume
+//
+//  Created by Hanzheng Li on 3/9/22.
+//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct TabBarReader: UIViewControllerRepresentable {
+    var completion: (UITabBar) -> Void
+    private let proxy = ProxyController()
+
+    func makeUIViewController(context: UIViewControllerRepresentableContext<TabBarReader>) -> UIViewController {
+        proxy.completion = completion
+        return proxy
+    }
+
+    func updateUIViewController(_ uiViewController: UIViewController, context: UIViewControllerRepresentableContext<TabBarReader>) { }
```

Comment:
is this supposed to be empty? can we remove this if it is?

Reply:
Can't remove bc it's from the protocol